rest/rest-client/src/main/java/org/ow2/proactive/scheduler/rest/ISchedulerClient.java

```diff
@@ -124,6 +124,47 @@ JobId submit(File job)
     JobId submit(URL job)
             throws NotConnectedException, PermissionException, SubmissionClosedException, JobCreationException;
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Remove the redundant '!unknownSymbol!' thrown exception declaration(s). [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3ARedundantThrowsDeclarationCheck)